Userland/Libraries/LibAudio/MP3Tables.h

```diff
@@ -133,37 +133,20 @@ constexpr auto MakeMixedScaleFactorBandArray()
-};
+constexpr auto ScaleFactorBandShort32000 = MakeShortScaleFactorBandArray<Array<size_t, 13> { 4, 4, 4, 4, 6, 8, 12, 16, 20, 26, 34, 42, 12 }>();
+constexpr auto ScaleFactorBandShort44100 = MakeShortScaleFactorBandArray<Array<size_t, 13> { 4, 4, 4, 4, 6, 8, 10, 12, 14, 18, 22, 30, 56 }>();
+constexpr auto ScaleFactorBandShort48000 = MakeShortScaleFactorBandArray<Array<size_t, 13> { 4, 4, 4, 4, 6, 6, 10, 12, 14, 16, 20, 26, 66 }>();
 
-// ISO/IEC 11172-3 (Table B.8)
-HashMap<int, Array<ScaleFactorBand, 39>> ScaleFactorBandsMixed = {
-    { 32000,
-        MakeMixedScaleFactorBandArray<Array<size_t, 8> { 4, 4, 4, 4, 4, 4, 6, 6 }, Array<size_t, 10> { 4, 6, 8, 12, 16, 20, 26, 34, 42, 12 }>() },
-    { 44100,
-        MakeMixedScaleFactorBandArray<Array<size_t, 8> { 4, 4, 4, 4, 4, 4, 6, 6 }, Array<size_t, 10> { 4, 6, 8, 10, 12, 14, 18, 22, 30, 56 }>() },
-    { 48000,
-        MakeMixedScaleFactorBandArray<Array<size_t, 8> { 4, 4, 4, 4, 4, 4, 6, 6 }, Array<size_t, 10> { 4, 6, 6, 10, 12, 14, 16, 20, 26, 66 }>() },
-};
+constexpr auto ScaleFactorBandMixed32000 = MakeMixedScaleFactorBandArray<Array<size_t, 8> { 4, 4, 4, 4, 4, 4, 6, 6 }, Array<size_t, 10> { 4, 6, 8, 12, 16, 20, 26, 34, 42, 12 }>();
+constexpr auto ScaleFactorBandMixed44100 = MakeMixedScaleFactorBandArray<Array<size_t, 8> { 4, 4, 4, 4, 4, 4, 6, 6 }, Array<size_t, 10> { 4, 6, 8, 10, 12, 14, 18, 22, 30, 56 }>();
+constexpr auto ScaleFactorBandMixed48000 = MakeMixedScaleFactorBandArray<Array<size_t, 8> { 4, 4, 4, 4, 4, 4, 6, 6 }, Array<size_t, 10> { 4, 6, 6, 10, 12, 14, 16, 20, 26, 66 }>();
 
-// ISO/IEC 11172-3 (Table B.8)
-HashMap<int, Array<ScaleFactorBand, 23>> ScaleFactorBandsLong = {
-    { 32000,
-        MakeLongScaleFactorBandArray<Array<size_t, 23> { 4, 4, 4, 4, 4, 4, 6, 6, 8, 10, 12, 16, 20, 24, 30, 38, 46, 56, 68, 84, 102, 25, 0 }>() },
-    { 44100,
-        MakeLongScaleFactorBandArray<Array<size_t, 23> { 4, 4, 4, 4, 4, 4, 6, 6, 8, 8, 10, 12, 16, 20, 24, 28, 34, 42, 50, 54, 76, 158, 0 }>() },
-    { 48000,
-        MakeLongScaleFactorBandArray<Array<size_t, 23> { 4, 4, 4, 4, 4, 4, 6, 6, 6, 8, 10, 12, 16, 18, 22, 28, 34, 40, 46, 54, 54, 192, 0 }>() },
-};
+constexpr auto ScaleFactorBandLong32000 = MakeLongScaleFactorBandArray<Array<size_t, 23> { 4, 4, 4, 4, 4, 4, 6, 6, 8, 10, 12, 16, 20, 24, 30, 38, 46, 56, 68, 84, 102, 26, 0 }>();
+constexpr auto ScaleFactorBandLong44100 = MakeLongScaleFactorBandArray<Array<size_t, 23> { 4, 4, 4, 4, 4, 4, 6, 6, 8, 8, 10, 12, 16, 20, 24, 28, 34, 42, 50, 54, 76, 158, 0 }>();
+constexpr auto ScaleFactorBandLong48000 = MakeLongScaleFactorBandArray<Array<size_t, 23> { 4, 4, 4, 4, 4, 4, 6, 6, 6, 8, 10, 12, 16, 18, 22, 28, 34, 40, 46, 54, 54, 192, 0 }>();
 
 // ISO/IEC 11172-3 (2.4.3.4.10.3 a)
-Array<double, 36> WindowBlockType0 { {
+Array<double, 36> WindowBlockTypeNormal { {
     AK::sin(AK::Pi<double> / 36 * (0 + 0.5)),
     AK::sin(AK::Pi<double> / 36 * (1 + 0.5)),
     AK::sin(AK::Pi<double> / 36 * (2 + 0.5)),
```

Comment:
Please move the fixes in this file to the commit that introduced the lookup tables in the first place.